recipe/build.sh

```diff
@@ -60,16 +138,38 @@ fi
         --enable-libaom \
         --enable-libsvtav1 \
         --enable-libxml2 \
-        --enable-libvpx \
         --enable-pic \
-        --enable-pthreads \
         --enable-shared \
         --disable-static \
         --enable-version3 \
         --enable-zlib \
-        --enable-libmp3lame \
-        --pkg-config=$BUILD_PREFIX/bin/pkg-config \
-        $EXTRA_CONFIGURE_OPTIONS || { cat ffbuild/config.log; exit 1; }
+        --pkg-config=${PKG_CONFIG} \
+        || { if [[ ${CI} != "" ]]; then cat ffbuild/config.log; fi; exit 1; }
+
+if [[ "${target_platform}" == win-* ]]; then
+  # Don't install the def files, but intall the lib files instead
+  sed -i 's/SLIB_INSTALL_EXTRA_LIB=$(SLIBNAME_WITH_MAJOR:$(SLIBSUF)=.def)/SLIB_INSTALL_EXTRA_LIB=$(SLIBNAME:$(SLIBSUF)=.lib)/' ffbuild/config.mak
+  # install the lib files in the lib directory, not bin
+  sed -i 's/SLIB_INSTALL_EXTRA_SHLIB=$(SLIBNAME:$(SLIBSUF)=.lib)/SLIB_INSTALL_EXTRA_SHLIB=/' ffbuild/config.mak
+
+  # Their default DLLs include the version in the dll name
+  # If we want to, we can uncomment the lines below to remove the version fro mthe dll name
+  # we don't need that because conda is our package manager
+  # sed -i 's/SLIB_INSTALL_NAME=$(SLIBNAME_WITH_MAJOR)/SLIB_INSTALL_NAME=$(SLIBNAME)/' ffbuild/config.mak
+  # sed -i 's/SLIBNAME_WITH_VERSION=$(SLIBPREF)$(FULLNAME)-$(LIBVERSION)$(SLIBSUF)/SLIBNAME_WITH_VERSION=$(SLIBNAME)/' ffbuild/config.mak
+  # sed -i 's/SLIBNAME_WITH_MAJOR=$(SLIBPREF)$(FULLNAME)-$(LIBMAJOR)$(SLIBSUF)/SLIBNAME_WITH_MAJOR=$(SLIBNAME)/' ffbuild/config.mak
```

Comment:
I don't mind much either way, but I have a slight preference to removing the version from the DLL, which should also allow to unify the testing of the libs into one loop.

Nit: "fro mthe" -> "from the"
Perhaps also: "we don't need that [...]" -> "we don't need those version numbers [...]"

Reply:
Thanks for catching this. Ultimately. I gave up because it was an uphill battle with many other libraries. it also somewhat acts as a buffer to detect library version changes upstream. so i don't really mind keeping the so version.

I find those sed commands to be hacky, though i figured i could leave them here as reference, at least for a bit.